### DISCUSSION TITLE
How do I debug a failed petsc installation?

### DISCUSSION THREAD

**Discussion by VictorEijkhout** [idaholab/moose#26983] [Q&A Getting Started]
```
*********************************************************************************************
           UNABLE to CONFIGURE with GIVEN OPTIONS (see configure.log for details):
---------------------------------------------------------------------------------------------
                            FC Compiler has no support for OpenMP
*********************************************************************************************

There was an error. Exiting...

CMake configuration ended: Tue Mar  5 12:25:43 CST 2024
```

1. Do I need Fortran?
2. I'm pretty sure my fortran compiler *does* support OpenMP. It's whatever comes with Intel19.
3. For starters, where is the petsc log file? I guess I can just search for it.
4. Oh, how recent an intel compiler do you support? Intel 24 gave an error message about being unsupported?

**Comment by GiudGiud** [comment]
> Oh, how recent an intel compiler do you support? Intel 24 gave an error message about being unsupported?

None.
https://mooseframework.inl.gov/sqa/minimum_requirements.html
